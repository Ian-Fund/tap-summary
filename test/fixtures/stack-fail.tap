TAP version 13
# passing test
ok 1 should be equal
# main has value
ok 2 should be equivalent
# failing test
not ok 3 should be equal
  ---
    operator: equal
    expected: false
    actual:   true
    at: Test.tape (/home/thomas/code/personal/lindease/test/test.js:18:8)
    stack: |-
      Error: should be equal
          at Test.assert [as _assert] (/home/thomas/code/personal/lindease/node_modules/tape/lib/test.js:226:54)
          at Test.bound [as _assert] (/home/thomas/code/personal/lindease/node_modules/tape/lib/test.js:77:32)
          at Test.equal (/home/thomas/code/personal/lindease/node_modules/tape/lib/test.js:386:10)
          at Test.bound [as equal] (/home/thomas/code/personal/lindease/node_modules/tape/lib/test.js:77:32)
          at Test.tape (/home/thomas/code/personal/lindease/test/test.js:18:8)
          at Test.bound [as _cb] (/home/thomas/code/personal/lindease/node_modules/tape/lib/test.js:77:32)
          at Test.run (/home/thomas/code/personal/lindease/node_modules/tape/lib/test.js:96:10)
          at Test.bound [as run] (/home/thomas/code/personal/lindease/node_modules/tape/lib/test.js:77:32)
          at Immediate.next [as _onImmediate] (/home/thomas/code/personal/lindease/node_modules/tape/lib/results.js:75:19)
          at runCallback (timers.js:705:18)
  ...

1..3
# tests 3
# pass  2
# fail  1

