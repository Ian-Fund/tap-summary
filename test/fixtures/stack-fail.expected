
[33m# Tests
[0m
[G[32mâœ” passing test [pass: 1, fail: 0][0m
[G[32mâœ” main has value [pass: 1, fail: 0][0m
[G[31mâœ– failing test [pass: 0, fail: 1][0m

[33m# Summary
[0m
[36mplanned: 3[0m
[36massertions: 3[0m
[32mpass: 2[0m
[31mfail: 1[0m

[33m# Fails
[0m
[36m[4m## failing test[0m[0m
[31m  âœ– should be equal[0m
    operator: equal
    expected: false
    actual:   true
    at: Test.tape (/home/thomas/code/personal/lindease/test/test.js:18:8)
    stack: |-
Error: should be equal
at Test.assert [as _assert] (/home/thomas/code/personal/lindease/node_modules/tape/lib/test.js:226:54)
at Test.bound [as _assert] (/home/thomas/code/personal/lindease/node_modules/tape/lib/test.js:77:32)
at Test.equal (/home/thomas/code/personal/lindease/node_modules/tape/lib/test.js:386:10)
at Test.bound [as equal] (/home/thomas/code/personal/lindease/node_modules/tape/lib/test.js:77:32)
at Test.tape (/home/thomas/code/personal/lindease/test/test.js:18:8)
at Test.bound [as _cb] (/home/thomas/code/personal/lindease/node_modules/tape/lib/test.js:77:32)
at Test.run (/home/thomas/code/personal/lindease/node_modules/tape/lib/test.js:96:10)
at Test.bound [as run] (/home/thomas/code/personal/lindease/node_modules/tape/lib/test.js:77:32)
at Immediate.next [as _onImmediate] (/home/thomas/code/personal/lindease/node_modules/tape/lib/results.js:75:19)

